From d854374792e8530acce9c53fbaae0da7e7f8f3c3 Mon Sep 17 00:00:00 2001
From: Kito Cheng <kito.cheng@sifive.com>
Date: Fri, 6 Nov 2020 11:05:50 +0800
Subject: [PATCH 22/60] RISC-V: Mark non-export symbol static and const in
 riscv-common.c

gcc/ChangeLog:

	* common/config/riscv/riscv-common.c (riscv_implied_info):
	Add static and const.
	(riscv_subset_list::handle_implied_ext): Add const due to
	riscv_implied_info changed to const.
---
 gcc/common/config/riscv/riscv-common.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gcc/common/config/riscv/riscv-common.c b/gcc/common/config/riscv/riscv-common.c
index 84695a6ed1b..9a576eb689b 100644
--- a/gcc/common/config/riscv/riscv-common.c
+++ b/gcc/common/config/riscv/riscv-common.c
@@ -54,7 +54,7 @@ struct riscv_implied_info_t
 };
 
 /* Implied ISA info, must end with NULL sentinel.  */
-riscv_implied_info_t riscv_implied_info[] =
+static const riscv_implied_info_t riscv_implied_info[] =
 {
   {"d", "f"},
   {NULL, NULL}
@@ -459,7 +459,7 @@ riscv_subset_list::handle_implied_ext (const char *ext,
 				       int minor_version,
 				       bool explicit_version_p)
 {
-  riscv_implied_info_t *implied_info;
+  const riscv_implied_info_t *implied_info;
   for (implied_info = &riscv_implied_info[0];
        implied_info->ext;
        ++implied_info)
-- 
2.25.1

